model_compression_toolkit/qat/keras/quantizer/quant_utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow as tf
+from typing import Tuple
+
+
+def adjust_range_to_include_zero(range_min: tf.Tensor,
+                                 range_max: tf.Tensor,
+                                 n_bits: int) -> Tuple[tf.Tensor, tf.Tensor]:
+    """
+    Adjusting the quantization range to include representation of 0.0 in the quantization grid.
+    If quantization per-channel, then range_min and range_max should be tensors in the specific shape that allows
```

Comment:
there's no channel axis here, so maybe only write that for per_channel quantization min\max should be a vector